eoxserver/resources/coverages/tests.py

```diff
@@ -449,18 +472,45 @@ def test_eoom_reader(self):
         reader = eoom.EOOMFormatReader(env)
         self.assertTrue(reader.test(xml))
         values = reader.read(xml)
-
         expected = {
             "identifier": "MER_FRS_1PNPDE20060816_090929_000001972050_00222_23322_0058_uint16_reduced_compressed",
-            "begin_time": datetime(2006, 8, 16, 9, 9, 29, tzinfo=utc),
-            "end_time": datetime(2006, 8, 16, 9, 12, 46, tzinfo=utc),
+            "begin_time": datetime(2006, 8, 16, 9, 9, 29, tzinfo=tzlocal()),
+            "end_time": datetime(2006, 8, 16, 9, 12, 46, tzinfo=tzlocal()),
```

Comment:
This should stay `utc`. The local timezone is insignificant, we expect all data to be in UTC, this is the only way to keep sane when dealing with worldwide EO-data.